project.clj

```diff
@@ -3,12 +3,37 @@
   :url "https://github.com/fredericksgary/test.chuck"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.6.0"]
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [org.clojure/clojurescript "0.0-3308"]
```

Comment:
what about `:scope "provided"`? core.logic does it [that way](https://github.com/clojure/core.logic/blob/master/project.clj#L11). It means cljs jar will not be included in test.chuck uberjar. I'm not sure if it also means it won't be installed as a transitive dependency, but I think so. I'll check that now, and also bump to cljs 1.7.xx